decentraland/unity-explorer
# Avatar fade dithering

## What does this PR change?

https://github.com/decentraland/unity-explorer/issues/734
Now avatars get gradually transparent as the camera gets closer to them, using a dithering effect. This effect does not affect elements added by emotes.

## How to test the changes?

1. Launch the explorer
2. Move the camera closer to the player's avatar.

The avatar should be fully opaque when camera is far from it and almost transparent when the camera is as close as possible.

1. Launch the explorer
2. Change the camera mode to first person.

The player's avatar should not be visible at all.

1. Launch the explorer
4. Move the player's avatar next to another avatar.
5. Rotate the camera so it gets closer to the other avatar.

The other avatar should be fully opaque when camera is far from it and transparent when the camera is as close as possible.

Note: This feature should not affect the avatar in the backpack or the loading screen.

https://github.com/user-attachments/assets/1edcc9f9-cd4f-4f19-b9fa-d8db470b7092
